### ISSUE TITLE
Remove getters for optional stage inputes

### ISSUE THREAD

**Issue by jlvdb** [LSSTDESC/rail_yaw#13]
Currently `YawRailStage` implements three methods to safely get and set stage inputs that are optional:
```python
YawRailStage.get_optional_handle
YawRailStage.get_optional_data
YawRailStage.set_optional_data
```
It should be possible to remove these and instead implement them using the `allow_missing=True`.

**Comment by jlvdb** [comment]
It seems that the current solution is already the best approach unless there are some changes made to how `ceci` handles and parses omitted parameters.

**Comment by jlvdb** [comment]
This was addressed in #18 where it was established that the optional getters are useful convenience methods
